specs/jsonschema-validation.md

```diff
@@ -328,10 +328,10 @@ an annotation.
 
 Implementations:
 
-- SHOULD provide validation for each format attribute defined in this
-  document;
-- MAY support format values not defined in this document, but such support MUST
-  be configurable and disabled by default;
+- SHOULD provide validation for each format attribute defined in this document
+  and listed in the {{format-registry}};
+- MAY support format values not defined in this document or listed in the
+  registry, but such support MUST be configurable and disabled by default;
 - SHOULD use a common parsing library or a well-known regular expression for
   each format;
 - SHOULD clearly document how and to what degree each format attribute is
```

Comment:
I don't think this bullet point makes sense. We don't allow partial validation anymore, so full validation is the only valid way to validate a format.

Reply:
I agree, but realistically implementations are going to support formats only as far as they care to, even if we include a bunch of edge-case tests.  I think having a note that requires implementations to declare their support (or maybe their limitations) covers that base.

Reply:
I see what you're saying, but I feel like it gives permission for incomplete validation, which we don't want.

> (or maybe their limitations) 

Yeah, that. I think that's better. You shouldn't have to say you support what you're required to support. You should only have declare when your support is lacking in some way.

Reply:
Updated